Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1481 @@
+    issue_id = args.get('issue_id', '')
+    comment_id = args.get('comment_id', '')
+    content = args.get('content', '')
+    body = {
+        'content': {
+            'raw': content
+        }
+    }
+    if not repo:
+        repo = client.repository
+    response = client.issue_comment_update_request(repo, issue_id, comment_id, body)
+    return CommandResults(readable_output='The comment was updated successfully',
+                          outputs_prefix='Bitbucket.IssueComment',
+                          outputs=response,
+                          raw_response=response)
+
+
+def issue_comment_list_command(client: Client, args: Dict) -> CommandResults:
+    """ Returns a list of comments.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a list of the comments or a single comment on a specific issue.
+    """
+    repo = args.get('repo')
+    issue_id = args.get('issue_id', '')
+    comment_id = args.get('comment_id', None)
+    limit = int(args.get('limit', 50))
+    page = int(args.get('page', 1))
+    check_args(limit, page)
+    page_size = min(100, limit)
+    params = {
+        'page': page,
+        'pagelen': page_size
+    }
+    if not repo:
+        repo = client.repository
+    response = client.issue_comment_list_request(repo, issue_id, comment_id, params)
+    if comment_id:
+        results = [response]
+        hr_title = f'The information about the comment "{comment_id}"'
+    else:
+        results = check_pagination(client, response, limit)
+        hr_title = f'List of the comments on issue "{issue_id}"'
+    human_readable = []
+    for value in results:
+        d = {'Id': value.get('id'),
+             'Content': value.get('content', {}).get('raw'),
+             'CreatedBy': value.get('user', {}).get('display_name'),
+             'CreatedAt': value.get('created_on'),
+             'UpdatedAt': value.get('updated_on'),
+             'IssueId': value.get('issue', {}).get('id'),
+             'IssueTitle': value.get('issue', {}).get('title')}
+        human_readable.append(d)
+
+    headers = ['Id', 'Content', 'CreatedBy', 'CreatedAt', 'UpdatedAt', 'IssueId', 'IssueTitle']
+    readable_output = tableToMarkdown(
+        name=hr_title,
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Bitbucket.IssueComment',
+        outputs=results,
+        raw_response=results
+    )
+
+
+def pull_request_comment_create_command(client: Client, args: Dict) -> CommandResults:
+    """ Creates a comment in a pull request.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a success message, in case of a successful creation of a comment.
+    """
+    repo = args.get('repo')
+    pr_id = int(args.get('pull_request_id', ''))
```

Comment:
Use arg_to_number

Reply:
I checked, and arg_to_number also validate the arguments, so I deleted the function check_args, and are using arg_to_number in all the relevant places.